torchaudio/csrc/sox_io.h

```diff
@@ -28,7 +28,8 @@ c10::intrusive_ptr<torchaudio::sox_utils::TensorSignal> load_audio_file(
     const int64_t frame_offset = 0,
     const int64_t num_frames = -1,
     const bool normalize = true,
-    const bool channels_first = true);
+    const bool channels_first = true,
+    const int64_t sample_rate = -1);
```

Comment:
Where is the value of -1 documented?

Reply:
It is omitted because I judged that the code is self-explanatory.